Lesson16/src/main.js

```diff
@@ -1,32 +1,42 @@
 import { format, differenceInCalendarDays } from "date-fns";
+const tabMenuList = document.getElementById("js-tab-menu__list");
 const API = {
-  news: "https://jsondata.okiba.me/v1/json/4N9rO211111110723",
-  book: "https://jsondata.okiba.me/v1/json/P40aO211111111010",
-  travel: "https://jsondata.okiba.me/v1/json/EB0MC211111111223",
-  economy: "https://jsondata.okiba.me/v1/json/TqU0F211111111341"
+  news: "./json/news.json",
+  book: "./json/book.json",
+  travel: "./json/travel.json",
+  economy: "./json/economy.json"
 };
-const tabMenuList = document.getElementById("js-tab-menu__list");
+const articleAPI = {
+  "8b5ae244-cddb-4b94-a5cd-b1ca4201c945": "./json/article-1.json",
+  "8b5ae244-cddb-4b94-a5cd-b1ca4201c946": "./json/article-2.json",
+  "7cb39430-d000-4b2a-8bbd-29a6ff6c7694": "./json/article-3.json",
+  "7cb39430-d000-4b2a-8bbd-29a6ff6c7697": "./json/article-4.json",
+  "4c42eb6a-dc41-4138-ac0a-49062e4a55e3": "./json/article-5.json",
+  "9d50f525-b5a3-469d-ad28-0bbaa3f38fcc": "./json/article-6.json",
+  "9d50f525-b5a3-469d-ad28-0bbaa3f38fce": "./json/article-7.json"
+}
 
-function addLoading() {
-  const tabContent = document.getElementById("js-tab-content");
-  const loading = document.createElement("img");
-  loading.src = "./loading-circle.gif";
-  loading.classList.add("loading");
-  loading.id = "loading";
-  tabContent.appendChild(loading);
+function createElementWithClass(type, name) {
```

Comment:
jsの文脈だとclassはインスタンスを作れるオブジェクトもあるので
```suggestion
function createElementWithClassName(type, name) {
```

Reply:
Thank you.
I fixed it.
4d19bcd